### ISSUE TITLE
Integrated file sharing for iPad

### ISSUE THREAD

**Issue by acertain** [keybase/client#10167]
I would really like to be able to go into an iPad app (e.g. Goodnotes), hit "Export" and have KeyBase show up as a target, so that I can export directly to a folder there. My use case is signing financial documents (signing like with my Apple Pencil and my hand-written signature), and I need a way to sign it and share the document back. I would love to do it through KeyBase. Obviously, first you need to support files at all on iPad, but I wanted to make sure that when you do, it's well-integrated into the iOS ecosystem so that apps can see it as a target for export (and you need to be able to open files in KeyBase and send them to the appropriate app as well).

I assume that this feature is already on your backlog somewhere, so if there's some other place I can just +1 that feature and watch progress, please let me know (I did search the github issues but didn't find anything).

**Comment by drcongo** [comment]
@acertain I think this issue is the one to follow, though there maybe an even earlier one also: https://github.com/keybase/client/issues/10007

**Comment by permezel** [comment]
I agree. The current share with facility does nothing useful. Needs more work and debug.  Need help?
